packages/core/data-transfer/lib/providers/local-strapi-destination-provider/index.ts

```diff
@@ -92,6 +94,49 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
     return mapSchemasValues(schemas);
   }
 
+  getAssetsStream(): NodeJS.WritableStream {
+    if (!this.strapi) {
+      throw new Error('Not able to stream Assets. Strapi instance not found');
+    }
+
+    const assetsDirectory = path.join(this.strapi.dirs.static.public, 'uploads');
+    const backupDirectory = path.join(
+      this.strapi.dirs.static.public,
+      `uploads_backup_${Date.now()}`
+    );
+
+    fse.renameSync(assetsDirectory, backupDirectory);
+    fse.mkdirSync(assetsDirectory);
+
+    return new Writable({
+      objectMode: true,
+      async final(next) {
+        fse.rmSync(backupDirectory, { recursive: true, force: true });
+        next();
+      },
+      async write(chunk, _encoding, callback) {
```

Comment:
```suggestion
      async write(chunk: IAsset, _encoding, callback) {
```

Reply:
If we make this typing, does it mean that we need to change the local-file-source-provider streamAssets method to:
```outStream.write({ filename: file, path: filePath, stats: { size }, stream: entry });```
instead of
```outStream.write({ file, path: filePath, stats: { size }, stream: entry });```

to match the IAsset interface:
```
export interface IAsset {
  filename: string;
  filepath: string;
  stream: NodeJS.ReadableStream;
  stats: IAssetStats;
}
```